stage0/src/acpi.rs

```diff
@@ -160,6 +162,12 @@ pub struct DescriptionHeader {
 }
 
 impl DescriptionHeader {
+    #[allow(dead_code)]
+    pub fn signature(&self) -> &str {
+        // Safety: per the ACPI spec, the signature is in ASCII, which is always valid UTF-8.
+        unsafe { core::str::from_utf8_unchecked(&self.signature) }
+    }
```

Comment:
should we make use the safe version, with `.expect("Safety: per the ACPI spec, the signature is in ASCII, which is always valid UTF-8")` instead? :) 

It'd seem that crashing is desired if the sig is not spec compliant

Reply:
Good point. Done.